packages/lib/storybook/public/mockServiceWorker.js

```diff
@@ -0,0 +1,302 @@
+/* eslint-disable */
+/* tslint:disable */
+
+/**
+ * Mock Service Worker.
+ * @see https://github.com/mswjs/msw
+ * - Please do NOT modify this file.
+ * - Please do NOT serve this file on production.
+ */
+
+const PACKAGE_VERSION = '2.7.0';
```

Comment:
out of curiosity - what package does this refer to? it does not seem to be related to the version that you installed in the `package.json`. 
do we need to keep track of this version when upgrading the msw package?

Reply:
Not sure, this is auto generated via `npx msw init` https://mswjs.io/docs/cli/init/. I don't believe it's something we need to manually updated.